setup.py

```diff
@@ -100,19 +91,11 @@ def emit(self, record):
             _unittests_only(suite, mod.test_suite())
     return suite
 
-doclines = __doc__.split("\n")
-
-def read_file(*path):
-    base_dir = os.path.dirname(__file__)
-    file_path = (base_dir, ) + tuple(path)
-    with open(os.path.join(*file_path), 'rb') as file:
-        return file.read()
-
-long_description = str(
-    ("\n".join(doclines[2:]) + "\n\n" +
-     ".. contents::\n\n" +
-     read_file("README.rst").decode('latin-1')  + "\n\n" +
-     read_file("CHANGES.rst").decode('latin-1')))
+long_description = (
+    open("README.rst").read()  +
```

Comment:
There is no leak here.  If this script was long lived, the files would eventually be closed by the garbage collector. Even if this was long lived, which it isn't, there is only a potential problem if the program were opening files faster than the GC could reap them.

The pedantic is the enemy of the productive.